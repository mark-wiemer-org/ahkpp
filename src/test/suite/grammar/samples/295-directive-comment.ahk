; [Issue #295](https://github.com/mark-wiemer/vscode-autohotkey-plus-plus/issues/295)
#KeyHistory
#KeyHistory MaxEvents
#KeyHistory MaxEvents ;Comment
#KeyHistory MaxEvents;NotComment
