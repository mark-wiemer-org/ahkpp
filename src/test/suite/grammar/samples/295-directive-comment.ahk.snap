>; [Issue #295](https://github.com/mark-wiemer/vscode-autohotkey-plus-plus/issues/295)
#^ source.ahk comment.line.semicolon.ahk punctuation.definition.comment.ahk
# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk comment.line.semicolon.ahk
>#KeyHistory
#^^^^^^^^^^^ source.ahk keyword.control.directives.ahk
>#KeyHistory MaxEvents
#^^^^^^^^^^^ source.ahk keyword.control.directives.ahk
#           ^^^^^^^^^^ source.ahk keyword.control.directives.ahk string.parameter.directives.ahk
>#KeyHistory MaxEvents ;Comment
#^^^^^^^^^^^ source.ahk keyword.control.directives.ahk
#           ^^^^^^^^^^ source.ahk keyword.control.directives.ahk string.parameter.directives.ahk
#                     ^^^^^^^^^ source.ahk keyword.control.directives.ahk comment.line.semicolon.directive.ahk
>#KeyHistory MaxEvents;NotComment
#^^^^^^^^^^^ source.ahk keyword.control.directives.ahk
#           ^^^^^^^^^^^^^^^^^^^^^ source.ahk keyword.control.directives.ahk string.parameter.directives.ahk
>